dcpy/connectors/edm/recipes.py

```diff
@@ -130,12 +130,9 @@ def import_dataset(
     preprocessor: Callable[[str, pd.DataFrame], pd.DataFrame] | None = None,
 ):
     """Import a recipe to local data library folder and build engine."""
-    if ds.file_type is None:
```

Comment:
Let's maybe throw an error in this case now?

Reply:
true, I added it to `import_dataset` where we already throw an error if version is unresolved